docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -94,63 +94,6 @@ def shopping_list() -> None:
     execute_query("CREATE TABLE shopping_list AS SELECT * FROM sugary_cereals")
 ```
 
-#### Defining basic managed-loading dependencies
```

Comment:
This content should likely be moved to the I/O manager concept page

Reply:
If we're removing this section, the links in lines 72-75 will have to be updated as well